src/components/Login.tsx

```diff
@@ -16,6 +16,10 @@ export const Login = ({ onLoginSuccess }: any) => {
     const [passwordEntered, setPasswordEntered] = useState('' as string)
     const [userpassEnabled, setUserpassEnabled] = useState(false as boolean)
 
+    const [selectedVO, setSelectedVO] = useState('def' as string)
+    const possibleVos = env('long_vos')?.split(',')
```

Comment:
In this approach, we assume that the same logical VO appears at the same index in both the `short_vos` and  `long_vos` CSV's. This creates a `very easy to overlook and potentially difficult to debug` configuration scenario, where a small oversight during the setup of the webapp may cause cross-vo login requests.

The environment variable way to solve this would be to allow the configuration of the webapp as follows:

```
# file:  .env
[multi_vo]
REACT_APP_multi_vo_enabled = true
REACT_APP_vos = atl, cms, abc, XYZ
REACT_APP_vo_atl = Atlas
REACT_APP_vo_cms = CMS
REACT_APP_vo_abc = A Babass Collider
```
Then somewhere in the Login.tsx file, we should
- check if multi_vo is enabled
- If true, parse the 'short_vos' environment variable
- For each short vo, find the vo_{short_vo} environment variable
...
I hope this one is clear enough, please let me know otherwise.